crates/gossip/tests/initial_sync.rs

```diff
@@ -57,11 +57,11 @@ async fn two_new_agents_sync() {
         .await;
 
     // Then both agents should have reached full arc.
-    harness_1
-        .wait_for_full_arc_for_all(Duration::from_secs(5))
-        .await;
     harness_2
```

Comment:
In theory, harness 2 should be ready first because it comes online second and will boostrap and initiate right away. That's not guaranteed but it's what I'd expect most of the time. 

Either way, increase the time these get to be ready. Locally this test reliably finishes in 2.1s...